modules/core/site.js

```diff
@@ -1961,10 +1961,23 @@ var imap_smtp_edit_action = function(event) {
     }
 };
 
+
 var hasLeadingOrTrailingSpaces = function(str) {
     return str !== str.trim();
 };
 
+var sprintf = function(format, ...args) {
+    let i = 0;
+    return format.replace(/%([sd])/g, (match, type) => {
+        let arg = args[i++];
+        switch (type) {
+            case 's': return String(arg);
+            case 'd': return Number(arg);
+            default: return match;
+        }
+    });
+}
+
```

Comment:
I don't think this is necessary. If including variables in a string is all you need, you can use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).

Reply:
Hello @jacob-js . I removed the sprint function and used template literals. Please review